libs/components/datetime/src/lib/modules/datepicker/datepicker.component.ts

```diff
@@ -185,8 +195,12 @@ export class SkyDatepickerComponent implements OnDestroy, OnInit {
 
   #overlayKeyupListener: Subscription | undefined;
 
+  #hintTextResourceUnsubscribe = new Subject<void>();
```

Comment:
Can we use the normal `#ngUnsubscribe` for this? We will likely refactor these to use the new `takeUntilDestroyed` once it's out of developer preview and finding all of them will be easier if they're named the same thing.